features/issue.feature

```diff
@@ -31,6 +31,116 @@ Feature: hub issue
       """
     And the exit status should be 0
 
+  Scenario: Fetch issues not assigned to any milestone
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :milestone => "none"
+
+      json [
+        { :number => 102,
```

Comment:
Done. However, we're not sure, by this test alone, that the assert is actually called.